### DISCUSSION TITLE
Fokus on an Element for a screenshot.

### DISCUSSION THREAD

**Discussion by gladklo** [ui5-community/wdi5#631] [Q&A] [answered]
Im trying to take a screenshot of a <sap.viz.ui5.controls.VizFrame> in a <sap.suite.ui.commons.ChartContainer>. But the chart is so low in the view that i can´t see it in the screenshot.

Is there a possibility to manually scroll or focus on a specific element? Or maybe set the browser zoom so the chart fits in the screen?

**Comment by Siolto** [comment, marked as answer]
You can use the native WebdriverIO function [scrollIntoView](https://webdriver.io/docs/api/element/scrollIntoView/). You can probably do something like this:

```javascript
// identify
const vizFrame = await browser.asControl({...})

// get the WebdriverIO element
const vizFrameWebElement = await vizFramge.getWebElement()

// scroll to element
await vizFrameWebElement.scrollIntoView()
```
